src/test/java/com/box/sdk/BoxFileTest.java

```diff
@@ -1394,6 +1394,52 @@ public void testUploadNewVersionReturnsCorrectInfo() throws IOException {
         Assert.assertEquals(fileName, info.getName());
     }
 
+    @Test
+    @Category(UnitTest.class)
+    public void createSharedLinkWithPasswordSucceeds() throws IOException {
+        final String fileID = "1111";
+        final String password = "test1";
+        String result = "";
+
+        JsonObject innerObject = new JsonObject()
+                .add("password", password);
+
+        JsonObject sharedLinkObject = new JsonObject()
+                .add("shared_link", innerObject);
+
+        result = TestConfig.getFixture("BoxFile/CreateSharedLinkWithPassword201");
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.put(WireMock.urlPathEqualTo("/files/" + fileID))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withStatus(200)));
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.put(WireMock.urlPathEqualTo("/files/" + fileID))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(result)));
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.put(WireMock.urlPathEqualTo("/files/" + fileID))
```

Comment:
Does this test actually make three API calls to the same URL?  Can you add comments to describe which calls these correspond to?